### ISSUE TITLE
Update incorrect document names of OIG report documents

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#2215]
**What we're after:** Before we can go live with the new OIG page and OIG reports search, we need to check all of the reports to make sure that the names and documents match. There are a few instances where the name incorrectly labels the document. 

**Completion criteria:**
- [x] Move reports to wagtail
- [x] Update spreadsheet with new links, and correct names and documents (https://docs.google.com/spreadsheets/d/1ZIismo1y8mih7O4XoCTuyq13uKwtCp_Dz_RzMstOFQ4/edit#gid=0)
- [x] Follow-up with the OIG team when the reports are checked and updated

**Comment by JonellaCulmer** [comment]
@patphongs What do you think about moving all the reports off transition and onto wagtail?

**Comment by JonellaCulmer** [comment]
Updating the spreadsheet above with all the relevant docs. Color-coding based on the status of upload to wagtail and creation of document page. Green will identify updated document pages with correct name, document link, and classification. 
https://docs.google.com/spreadsheets/d/1ZIismo1y8mih7O4XoCTuyq13uKwtCp_Dz_RzMstOFQ4/edit#gid=0

**Comment by JonellaCulmer** [comment]
Updated the spreadsheet further with updated names and document links in wagtail. There are a few places where I do not have accurate information. Those are specifically the dates for some documents. Missing dates are highlighted in green. General dates were used in place. As part of OIG's review they must confirm and/or provide updated dates. 

Additionally, I identified some duplicate documents. Those are highlighted in red. 

https://docs.google.com/spreadsheets/d/1ZIismo1y8mih7O4XoCTuyq13uKwtCp_Dz_RzMstOFQ4/edit#gid=0
